### PR TITLE
Editorial: Move some members to App Information Note

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/pull/924.html" title="Last updated on Nov 17, 2020, 7:10 PM UTC (99eb8c3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/924/7e60981...99eb8c3.html" title="Last updated on Nov 17, 2020, 7:10 PM UTC (99eb8c3)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@aarongustafson, I've reverted your other commit from gh-pages and now your changes live in this branch :)

File: index.html [hunk 1]
```diff
@@ -2882,6 +2745,29 @@ <h2>
...
+      <ul>
+        <li>{{WebAppManifestAppInfo/categories}}
+        </li>
+        <li>{{WebAppManifestAppInfo/description}}
+        </li>
+        <li>{{WebAppManifestAppInfo/iarc_rating_id}}
+        </li>
+        <li>{{WebAppManifestAppInfo/screenshots}}
+        </li>
+      </ul>
```
Comment (marcoscaceres) [inline index.html:2769]:
```suggestion
      <ul>
        <li>`categories`
        </li>
        <li>`description`
        </li>
        <li>`iarc_rating_id`
        </li>
        <li>`screenshots`
        </li>
      </ul>
```

File: index.html [hunk 2]
```diff
@@ -2882,6 +2745,29 @@ <h2>
...
+      <h2>
+        Application Information
+      </h2>
+      <p>
+        Several members of the Web App Manifest provide additional metadata
+        related to how the web application may be presented in the context of a
+        digital storefront, installation dialog, or other surfaces where this
+        web application may be marketed or distributed. In an effort to support
+        these use cases better, the following members have been moved into
+        {{WebAppManifestAppInfo}}:
```
Comment (marcoscaceres) [inline index.html:2758]:
```suggestion
        [[[manifest-app-info]]]:
```

**Comment by marcoscaceres** [comment]
Blocked on publishing "manifest-app-info". Should be published next week - CFC ends 27th of July, 2020. https://github.com/w3c/manifest-app-info/issues/13

**Comment by aarongustafson** [comment]
@marcoscaceres I totally missed that this was blocked. App Info has published, can we push this now too?

**Pull request merged**